Apps/Sandcastle/gallery/Spheres and Ellipsoids.html

```diff
@@ -62,6 +62,178 @@
+        material : Cesium.Color.RED.withAlpha(0.3),
+        outline : true
+    }
+});
+
+viewer.entities.add({
+    name : 'Dome with top cut out',
+    position: Cesium.Cartesian3.fromDegrees(-108.0, 40.0),
+    ellipsoid : {
+        radii : new Cesium.Cartesian3(200000.0, 200000.0, 200000.0),
+        innerRadii : new Cesium.Cartesian3(100000.0, 100000.0, 100000.0),
+        minimumElevation : 0.0,
+        maximumElevation : Cesium.Math.toRadians(70.0),
+        material : Cesium.Color.YELLOW.withAlpha(0.3),
+        outline : true
+    }
+});
+
+viewer.entities.add({
+    name : 'Top and bottom cut out',
+    position: Cesium.Cartesian3.fromDegrees(-102.0, 40.0, 140000.0),
+    ellipsoid : {
+        radii : new Cesium.Cartesian3(200000.0, 200000.0, 200000.0),
+        innerRadii : new Cesium.Cartesian3(100000.0, 100000.0, 100000.0),
+        minimumElevation : Cesium.Math.toRadians(-40.0),
+        maximumElevation : Cesium.Math.toRadians(40.0),
+        material : Cesium.Color.DARKCYAN.withAlpha(0.3),
+        outline : true
+    }
+});
+
+viewer.entities.add({
+    name : 'Partial azimuth',
+    position: Cesium.Cartesian3.fromDegrees(-96.0, 39.5),
+    ellipsoid : {
+        radii : new Cesium.Cartesian3(200000.0, 200000.0, 200000.0),
+        innerRadii : new Cesium.Cartesian3(150000.0, 150000.0, 150000.0),
+        minimumAzimuth : Cesium.Math.toRadians(-90.0),
+        maximumAzimuth : Cesium.Math.toRadians(180.0),
+        minimumElevation : Cesium.Math.toRadians(20.0),
+        maximumElevation : Cesium.Math.toRadians(80.0),
+        material : Cesium.Color.GREEN.withAlpha(0.3),
+        outline : true
+    }
+});
+
+viewer.entities.add({
+    name : 'Upside down',
+    position: Cesium.Cartesian3.fromDegrees(-90.0, 39.0, 200000.0),
+    ellipsoid : {
+        radii : new Cesium.Cartesian3(200000.0, 200000.0, 200000.0),
+        innerRadii : new Cesium.Cartesian3(150000.0, 150000.0, 150000.0),
+        minimumAzimuth : Cesium.Math.toRadians(-90.0),
+        maximumAzimuth : Cesium.Math.toRadians(180.0),
+        minimumElevation : Cesium.Math.toRadians(-90.0),
+        maximumElevation : Cesium.Math.toRadians(20.0),
+        material : Cesium.Color.BLUE.withAlpha(0.3),
+        outline : true
+    }
+});
+
+viewer.entities.add({
+    name : 'Partial dome',
+    position: Cesium.Cartesian3.fromDegrees(-84.0, 38.5),
+    ellipsoid : {
+        radii : new Cesium.Cartesian3(200000.0, 200000.0, 200000.0),
+        minimumAzimuth : Cesium.Math.toRadians(-90.0),
+        maximumAzimuth : Cesium.Math.toRadians(180.0),
+        minimumElevation : 0.0,
+        material : Cesium.Color.RED.withAlpha(0.3),
+        outline : true
+    }
+});
+
+var airplanePosition = Cesium.Cartesian3.fromDegrees(-110.0, 35.0, 250000.0);
+viewer.entities.add({
+    id : 'airplane',
+    name : 'Airplane with sensor',
+    position : airplanePosition,
+    model : {
+        "../../SampleData/models/CesiumAir/Cesium_Air.gltf",
```

Comment:
This has a syntax error on load.
```
Uncaught SyntaxError: Unexpected token , (on line 134)
```

Reply:
There's also a few more like this in this file.

Reply:
Probably just remove the model entities from the example.

Reply:
@pjcozzi I'd really like to leave the model entities in the example, unless you feel strongly that they shouldn't be included. I think it would be very helpful to provide examples for how to combine a model with a partial ellipsoid.

Reply:
I really do not think they add value - keep the example focused.